cmd/runtimetest/main.go

```diff
@@ -370,6 +371,62 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func getActualCgroupPath() (string, error) {
+	f, err := os.Open("/proc/self/cgroup")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	cgroupPath := ""
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, ":")
+		if cgroupPath == parts[2] && cgroupPath != "" {
+			continue
+		} else if cgroupPath == "" && parts[2] != "" {
```

Comment:
is `parts[2] == ""` possible?  If we want to guard against it, I'd rather have a check like:

    if len(parts) != 3 || parts[2] == "" {
        return "", fmt.Errorf("invalid line in /proc/self/cgroup: %q", text)
    }

right after the `Split`.